src/bin/cargo/cli.rs

```diff
@@ -470,6 +472,7 @@ See 'cargo help <command>' for more information on a specific command.\n",
         .arg(flag("frozen", "Require Cargo.lock and cache are up to date").global(true))
         .arg(flag("locked", "Require Cargo.lock is up to date").global(true))
         .arg(flag("offline", "Run without accessing the network").global(true))
+        .arg(flag("absolute-paths", "Use absolute paths when calling rust").global(true))
```

Comment:
I am not sure. Should this be global? Some commands might not need it. Or maybe it controls not only "rustc invocations" but also all output related to filesystem paths.

Reply:
Also, I feel like this should be a `-Z` flag (unstable flag) first.